src/controller/CommissioningWindowOpener.h

```diff
@@ -107,6 +107,39 @@ class CommissioningWindowOpener
                                        Callback::Callback<OnOpenCommissioningWindow> * callback, SetupPayload & payload,
                                        bool readVIDPIDAttributes = false);
 
+    /**
+     * @brief
+     *   Try to look up the bridged device on the bridge with the given node id
```

Comment:
This class currently does not seem to reference bridges at all.

Could we re-phrase the documentation explaining that this is for use for bridges?